test/e2e/framework/scaletest/templates/kapinger-rbac.go

```diff
@@ -0,0 +1,45 @@
+package templates
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	KapingerClusterRole = rbacv1.ClusterRole{
```

Comment:
+same for this, let's stick to one way of deploying kapinger and tweak that if we need to

Reply:
Done